### ISSUE TITLE
Taxonomy

### ISSUE THREAD

**Issue by vendethiel** [vendethiel/projects#80]
**Meta**: For these issues describing the repo.

**Birthing**: Project created, doesn't do much at all yet
**Running**: I work on these regularly
**99%**: Project is almost done. Usually needs uploading/typo check/slightly better UX
**Done**: We did it! Yay!

**Waiting on external stuff**: A library/lang I want to use isn't ready/is buggy ATM, will be revisited
**Sleeping**: I want to get to the project, but not now
**Frozen**: Project hasn't been touched in several years, they're the lowest priority
**Deprecated**: This project is not useful anymore -- there are technically still things that *could* be done, but I won't
